docs/policy/software-release.md

```diff
@@ -42,24 +50,20 @@ However, there are important caveats about the version number:
 1.  The version number is only meaningful in the *osg-release* repository, though for technical reasons it's present (as
     an RPM) in other repositories.
 
-The version number is communicated in two ways:
+The version number is communicated as follows:
 
 1.  Every time a new release is made, the version number is updated.
     All release notes and communication to users about this release uses the new version number.
-1.  There is an *osg-version* RPM that reports the version of the release. Major metapackages (osg-ce, osg-client,
-    etc...) depend on this RPM.
-    The RPM itself has the version number in it. It also provide a program that reports the version, and a text file
-    that contains the version number.
 
-The version number will be of the form X.Y.Z. As of this writing, version numbers are 3.4.Z, where Z indicates a minor
+The version number will be of the form X.Y.Z. As of this writing, version numbers are 3.5.Z, where Z indicates a minor
 revision.
 
 Progression of Repositories
 ---------------------------
 
 This figure shows the progression of repositories that packages will go through:
 
-     osg-development -> osg-testing -> osg-prerelease / osg-rolling -> osg-release
+     osg-development -> osg-testing -> osg-prerelease -> osg-release
```

Comment:
Since we communicate above that 3.6 doesn't have a rolling repo, I think we can keep this

Reply:
Updated as requested